applications/API/urls.py

```
@@ -0,0 +1,21 @@
+from django.urls import path
+from drf_spectacular.views import (
+    SpectacularAPIView,
+    SpectacularRedocView,
+    SpectacularSwaggerView,
+)
+from rest_framework.routers import DefaultRouter
+from django.templatetags.static import static
```

Ruff (0.12.2)

`django.templatetags.static.static` imported but unused

Remove unused import: `django.templatetags.static.static`

(F401)